java/shared/pom.xml

```diff
@@ -59,7 +59,15 @@
     <dependency>
       <groupId>org.bitcoinj</groupId>
       <artifactId>bitcoinj-core</artifactId>
-      <version>0.15.10</version>
+      <version>0.16</version>
```

Comment:
![Screen Shot 2021-11-15 at 12 17 42 PM](https://user-images.githubusercontent.com/5118686/141862025-c85b3d6f-440c-4daa-84a4-3deeb8d0462a.png)
diff showing the introduction of protobuf-lite in 0.16 version of bitcoinj-core